### PR TITLE
Enable family planning form to be shown

### PR DESCRIPTION
**Pull request by makombe**
For some reason My IDE seems to have formatted common metadat file

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java [hunk 1]
```diff
@@ -269,119 +277,200 @@ public void install() {
...
+		if (Context.getAdministrationService()
+				.getGlobalPropertyObject(CommonMetadata.GP_CLIENT_VERIFICATION_USE_EMR_PROXY) == null) {
+			install(globalProperty(GP_CLIENT_VERIFICATION_USE_EMR_PROXY,
+					"Use the EMR backend to proxy NUPI requests (true or false)", "false"));
+		}
+		if (Context.getAdministrationService()
+				.getGlobalPropertyObject(CommonMetadata.GP_CLIENT_VERIFICATION_EMR_VERIFICATION_PROXY_URL) == null) {
+			install(globalProperty(GP_CLIENT_VERIFICATION_EMR_VERIFICATION_PROXY_URL,
+					"The local EMR URL to proxy NUPI verification requests",
+					"http://127.0.0.1:8080/openmrs/ws/rest/v1/kenyaemr/verifynupi"));
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java:369]: This won't work. Localhost can't be used. Also, let's configure these

Comment (makombe) [reply api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java:369]: @ojwanganto This has been there, did not do any change on global properties. Only added form and encounter type metadata for family planning. My IDEA seems to be formatting files

Comment (makombe) [reply api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java:369]: Maybe we would need to clean such in a followup PR.

**Review by njorocs — APPROVED** [review]
Family planning feautre lgtm. Others are IDE formatting changes which can be ignored.

File: api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java (hunk 1 above)
Comment (njorocs) [reply api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java]: @makombe , consider the suggestion from @ojwanganto. Merging this work for testing

**Review by PatrickWaweru — COMMENTED** [review]
Looks Good

**Pull request merged**